app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java

```diff
@@ -0,0 +1,150 @@
+package devs.erasmus.epills.utils;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import devs.erasmus.epills.broadcast_receiver.AlarmBroadcastReceiver;
+
+/**
+ * Created by Remo Andreoli on 15/11/2017.
+ */
+
+public class SetAlarmUtil_DEPRECATED {
```

Comment:
If it's deprecated - why keep it?

Reply:
i will delete it later, just in case i will re-use it

Reply:
but now that i think about it the new SetAlarmUtil seems stable, so i will do it now